microsoft/testsuites/network/sriov.py

```diff
@@ -542,6 +544,158 @@ def sriov_ethtool_offload_setting(self, environment: Environment) -> None:
         )
         assert_that(iperf_log).does_not_contain("error")
 
+    @TestCaseMetadata(
+        description="""
+        This case is to verify interrupts count increased after network traiffic
+         went through the VFs.
+
+        Steps,
+        1. Start iperf3 on server node.
+        2. Get initial interrupts sum per irq and cpu number on client node.
+        3. Start iperf3 for 5 minutes with 128 threads on client node.
+        4. Get final interrupts sum per irq number on client node.
+        5. Compare interrupts changes, expected to see interrupts increased.
+        6. Get final interrupts sum per cpu on client node.
+        7. Compare interrupts changes, expected to see interrupts increased.
+        """,
+        priority=2,
+        timeout=1200,
+        requirement=node_requirement(
+            node=schema.NodeSpace(
+                node_count=2,
+                core_count=search_space.IntRange(min=8, max=16),
+                network_interface=schema.NetworkInterfaceOptionSettings(
+                    nic_count=search_space.IntRange(min=1, max=8),
```

Comment:
This rule is not necessary. It almost allows all nic count.

Reply:
removed